webcomponents/ws-date-picker/ws-date-picker.scss

```diff
@@ -0,0 +1,211 @@
+$export-form-css: false;
```

Comment:
This file is definitely in the wrong folder. The folder webcomponents doesn't exists any more